host_core/config/config.exs

```diff
@@ -21,4 +21,7 @@ config :opentelemetry,
   span_processor: :batch,
   traces_exporter: :none
 
+config :host_core,
+  retain_rpc_subscriptions: false
```

Comment:
Wouldn't we want the default to be `true` or am I misreading/misinterpreting the code?

Reply:
default should be `false`. You only ever want to do this retention in test. I should probably document the ramifications of this in the config file(s)

Reply:
Yeah, documenting that sounds like a good idea